packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -421,11 +445,31 @@ export default class Calendar extends React.Component {
       return
     }
     return (
-      <div
+      <button
         className={cx('react-datepicker-today-button')}
-        onClick={e => this.props.onSelect(getStartOfDate(now(this.props.utcOffset)), e)}>
+        onClick={e => this.props.onSelect(getStartOfDate(now(this.props.utcOffset)), e)}
+      >
         {this.props.todayButton}
-      </div>
+      </button>
+    )
+  }
+
+  renderCloseButton = () => {
+    if (!this.props.todayButton) {
```

Comment:
What's the reason for rendering the hidden close button only when there is no Today button?

Reply:
copy/paste error I made from copying the renderTodayButton method. Fixed in this commit: https://github.com/cerner/terra-framework/pull/866/commits/7d90f03bdb30812bc8efd3b14705c40722d85d70